src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -52,13 +52,31 @@ public struct WarningEntry
             public int AstID;
             public string Filename;
         }
+
+        internal enum InfoID
+        {
+            Default
+        }
+
+        public struct InfoEntry
+        {
+            internal Runtime.InfoID ID;
+            internal string Message;
+            internal int Line;
+            internal int Column;
```

Comment:
I don't think we need Line and Column. These were originally used for warnings on a per line basis for code block node scripts.